README.rst

```diff
@@ -18,7 +18,7 @@ SQVID
    :alt: License Status
 
 
-SQVID, the Simple sQl Validator of varIous Datasources is a framework for
+SQVID, the Simple sQl Validator of various Datasources is a framework for
```

Comment:
@Adman Strange as it may sound, the `I` here is actually intentional, as it helps us generate the actual `SQVID` acronym (i.e. it stands for the `I` there).

If you would not mind, I would really like to keep it in.

Thanks!